tests/notifier/notifiers/cscc_notifier_test.py

```diff
@@ -21,25 +21,25 @@
 from google.cloud.forseti.notifier.notifiers import cscc_notifier
 from google.cloud.forseti.services.scanner import dao as scanner_dao
 from tests.services.scanner import scanner_dao_test
-from tests.unittest_utils import ForsetiTestCase
 
-class CsccNotifierTest(ForsetiTestCase):
+class CsccNotifierTest(scanner_dao_test.DatabaseTest):
```

Comment:
The naming here can be improved, as a test should not subclass from another test.  So, you should call this `BaseDbTestCase`.  Also, if this is something common, it should be moved to the `tests.unittest_util` package.

Reply:
That base class is not general purpose but tailored toward scanner testing -- I have thus renamed it but left it in the same place (`tests/services/scanner/scanner_dao_test.py`)